FittedSheetsPod/SheetViewController.swift

```diff
@@ -28,6 +28,13 @@ public class SheetViewController: UIViewController {
     /// If true, tapping on the overlay above the sheet will dismiss the sheet view controller
     public var dismissOnBackgroundTap: Bool = true
     
+    /// If true, sheet's dismiss view will be generated, otherwise sheet remains fixed and will need to be dismissed programatically
+    public var dismissable: Bool = true {
+        didSet {
+            guard isViewLoaded else { return }
```

Comment:
Everything looks good, but not sure why you have this `didSet` here. It doesn't seem to do anything